CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 Bugfixes:
+- Proxies using records derived from a base generic record broken using .NET 6 compiler (@CesarD, #632)
```

Comment:
Shouldn't this have your username and likely the pull request since you fixed it?

Reply:
I am generally quite happy to give credits to the person & issue reporting a problem, as opposed to my PR fixing it. So I did the same thing here that I did with #601:

https://github.com/castleproject/Core/blob/451c58fc9510c9faecaf5579fdafa8d65a53e978/CHANGELOG.md#L17

That being said, I'm actually unsure about the original intent behind the changelog's "bugfixes" section: should it reference the bug reports, or the PRs fixing them?

Reply:
No worries, I think I've included both the reporter (especially when they've provided a good repro) and the fixer before. There was a time when I only listed external contributors. Often bug fixes won't have an issue number, just the pull request.

Happy for you to decide what makes sense.

Reply:
OK, in that case I'll leave it as is... for now. Still undecided about what makes the _most_ sense, but it probably doesn't matter all that much, since links on GitHub pages make the relationships between issues and PRs easily discoverable.